README.md

```diff
@@ -1,32 +1,33 @@
-# Indeed Open Source Repository Template
+# Mariner
 
 ![OSS Lifecycle](https://img.shields.io/osslifecycle/indeedeng/default-template.svg)
 
 
-A default template repository we can use to bootstrap new open source projects. Replace this text with an overview of your project and what it does.
-
-Update the OSSLifeCycle shield above with the name of your repository (i.e. replace `default-template.svg` with `YOUR-REPOSITORY-NAME.svg`)
-
-Your README.md should contain the following sections:
+Mariner takes an input list of dependencies, fetches details about them from GitHub, 
+and outputs a file containing funding information for each project owner, and a list 
+of issues for each project
 
 ## Getting Started
 
-How does a user get started using this project?
+Run ```npm ci``` to install the libraries used in the project. Read more about [npm ci here.](https://blog.npmjs.org/post/171556855892/introducing-npm-ci-for-faster-more-reliable)
 
 ## Getting Help
 
-How does a user ask questions if they are stuck?
+The Open Source team at Indeed, who can be reached at opensource@indeed.com 
 
 ## How To Contribute
 
-What does a user need to know if they want to start contributing? If this information is extensive, capture it in a CONTRIBUTING.md file and link to that file here.
+Read the Code of Conduct and Contact the Maintainers before making any changes or a PR. 
+If an issue doesn’t already exist that describes the change you want to make, we recommend 
+creating one. If an issue does exist, please comment on it saying that you are starting to 
+work on it, to avoid duplicating effort. 
 
 ## Project Maintainers
 
-Who are the project maintainers, and how can they be reached?
+The Open Source team at Indeed, who can be reached at opensource@indeed.com
```

Comment:
We should include a link to our public web page: https://opensource.indeedeng.io/